lib/routes.dart

```diff
@@ -3,6 +3,8 @@ class Routes {
 
   static const ppe = '/ppe';
   static const ppeOnGuidance = '/ppe/on';
+  static const ppeOffGuidance1 = '/ppe/off/1';
```

Comment:
could we please make this a bit more descriptive, so instead of `/1` `method_1`

Reply:
Cleaned it up 4d9ce74 and 18ebb49